perfkitbenchmarker/benchmark_spec.py

```diff
@@ -175,6 +176,13 @@ def __init__(self, benchmark_info):
             not FLAGS['num_striped_disks'].present):
           num_striped_disks = (vm.max_local_disks /
```

Comment:
Also, we should probably use explicit integer floor division `//`, just to be safe when PerfKit moves to Python3